installer/roles/kubernetes/templates/deployment.yml.j2

```diff
@@ -134,6 +134,33 @@ spec:
     spec:
       serviceAccountName: awx
       terminationGracePeriodSeconds: 10
+{% if custom_venvs is defined %}
+      initContainers:
+        - image: 'centos:7'
+          name: init-custom-venvs
+          command:
+            - sh
+            - '-c'
+            - >-
+              yum install -y ansible python-pip curl python-setuptools epel-release openssl openssl-devel gcc python-devel &&
+              curl 'https://bootstrap.pypa.io/get-pip.py' | python &&
+              pip install virtualenv &&
+              mkdir -p {{ custom_venvs_path }} &&
+{% for custom_venv in custom_venvs %}
```

Comment:
@fantashley,

Thanks for automating this 😄

Is there any chance I could convince you to add one more thing to this?
What if `custom_venv.python_version` existed, and allowed you to conditionally create a py2 or py3 virtualenv?  A lot of users care about creating a custom venv because they want to run their playbooks under py3.

Reply:
@ryanpetrello,

My pleasure! I've added support for python3 as you requested. Just need to update the documentation and rebase and it should be good to go!